bootstrap/bootstrap.py

```diff
@@ -345,7 +348,8 @@ def serve_forever(server):
             ["apt-get", "install", "--yes", "software-properties-common"],
             env=apt_get_adjusted_env,
         )
-        run_subprocess(["add-apt-repository", "universe", "--yes"])
+        if distro == "ubuntu":
+            run_subprocess(["add-apt-repository", "universe", "--yes"])
```

Comment:
Can you clarify and inline a comment about why this is conditional to ubuntu?

Reply:
Universe section is specific to Ubuntu, so it cannot be added in Debian. And it is not needed there since all required packages are in main. Probably, the corresponding section in Debian would be "contrib".